src/Modules/FudgeTools.psm1

```diff
@@ -1742,13 +1754,17 @@ function Invoke-Chocolatey
     # set the parameters to pass
     $ParametersArg = Format-ChocolateyParams $Parameters
 
+    # if the version is latest, attempt to get the real current version
+    if (Test-VersionPassedIsLatest -Version $Version) {
+        $Version = Get-ChocolateyLatestVersion -Package $Package -Source $Source
+    }
```

Comment:
This is the main change; to detect if the version is using "latest", and get the current version from chocolatey